features/step_definitions/pull_request_steps.rb

```diff
@@ -0,0 +1,3 @@
+Then(/^I see a new (.+?) pull request for the "(.+?)" branch in the "(.+?)" repo in my browser$/) do |domain, branch, repo|
+  expect(@last_run_result.out).to eql "#{@tool} called with: #{pull_request_url domain, branch, repo}\n"
+end
```

Comment:
As per our discussion move this with the step about viewing a repository homepage into browser_helpers. Then :ship: